src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Microsoft.Extensions.Configuration.Binder.SourceGeneration.csproj

```
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisVersion_LatestVS)" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.7.0-3.23314.3" PrivateAssets="all" />
```

This is not going to be source-build compatible.  Why is the versions.props property no longer being utilized?  Source-build had [special logic](https://github.com/dotnet/installer/blob/main/src/SourceBuild/content/repo-projects/runtime.proj#L49) to lift this to the source-build version to prevent prebuilts.

---

```suggestion
    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisVersion_LatestVS)" PrivateAssets="all" />
    <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Condition="'$(MicrosoftCodeAnalysisVersion_LatestVS)' == '4.5.0'"  Version="4.7.0-3.23314.3" />
```

I wonder if we can do this to limp along until RC2.  This would let us build against the newer code analysis outside source build, but during sourcebuild when the newer version was specified we'd use that.

---

I'll proceed with this workaround for RC2 and stay tuned for further feedback.